libs/Makefile

```diff
@@ -1,37 +1,45 @@
-build:
```

Comment:
These changes seem unrelated to the task of building Idris with nix. Could they be in a separate PR/commit? And do they maintain compatibility with BSD make?

Reply:
It doesn't try to maintain compatibility with BSD make because Setup.hs already assumes GNU make (see `mymake` definition).

I can extract that to a separate PR if desired, but I'd prefer to keep it in this PR as well, since I had problems with missed dependencies until I changed the Makefiles.

Reply:
OK. @markuspf , is this likely to cause a problem for BSD support?